omnipod/src/main/res/values/strings.xml

```diff
@@ -255,6 +255,14 @@
     <string name="omnipod_less_than_a_minute_ago">Less than a minute ago</string>
     <string name="omnipod_composite_time">%1$s and %2$s</string>
     <string name="omnipod_time_ago">%1$s ago</string>
+    <string name="omnipod_cmd_beep_config">Beep config</string>
+    <string name="omnipod_cmd_play_test_beep">Play test beep</string>
+    <string name="key_omnipod_test_beep_button_enabled">AAPS.Omnipod.test_beep_button_enabled</string>
+    <string name="omnipod_config_test_beep_button_enabled">Test beep button enabled</string>
+    <string name="omnipod_overview_button_test_beep">Test beep</string>
+    <string name="omnipod_config_rileylink_stats_button_enabled">RileyLink Stats button enabled</string>
+    <string name="key_omnipod_rileylink_stats_button_enabled">AAPS.Omnipod.rileylink_stats_button_enabled</string>
```

Comment:
Can you please organize these string resources like the other ones? When you open the file, you will see that we organized them into different categories (mostly based on their prefix)

Reply:
This was tough. I hope I guessed them all correctly